src/vaillant_netatmo_api/thermostat.py

```diff
@@ -93,6 +95,65 @@ async def async_get_thermostats_data(self) -> list[Device]:
 
         return [Device(**device) for device in body["body"]["devices"]]
 
+    async def async_get_dhw_data(
+        self,
+        device_id: str,
+        home_id: str,
+    ) -> None:
```

Comment:
Return parameter type hint is wrong, this method _does_ return something.

Reply:
Oh yes. You are right